tensorboard/webapp/metrics/views/card_renderer/scalar_card_component.ts

```diff
@@ -257,10 +259,39 @@ export class ScalarCardComponent<Downloader> {
     return dataTableData;
   }
 
+  getMinMaxStepInSeries() {
```

Comment:
I can think about either (1) pass {minStep, maxStep} from container (2) put this in utils to be reused in container and component.
I might prefer (1) but at least I think we should have (2) to avoid same methods copied.
For (1) we can have 
```
type MinMapStep {
min: number
max: number 
}
```
and usage in line 285 will be `this.minMapStep.min`

Reply:
Ok I went with making a minMax observable(option 1). PTAL.